plugins/polio/js/src/domains/Notifications/Table/NotificationsTable.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { FunctionComponent } from 'react';
+import { Column, useSafeIntl } from 'bluesquare-components';
+
+import { DateCell } from '../../../../../../../hat/assets/js/apps/Iaso/components/Cells/DateTimeCell';
+import { TableWithDeepLink } from '../../../../../../../hat/assets/js/apps/Iaso/components/tables/TableWithDeepLink';
+import { handleTableDeepLink } from '../../../../../../../hat/assets/js/apps/Iaso/utils/table';
+
+import MESSAGES from '../messages';
+import { DeleteNotificationModal } from '../Modals/NotificationsDeleteModal';
+import { EditNotificationModal } from '../Modals/NotificationsCreateEditModal';
+import { NOTIFICATIONS_BASE_URL } from '../../../constants/routes';
+import { useGetNotifications } from '../hooks/api';
+import {
+    ApiNotificationsParams,
+    DropdownsContent,
+    NotificationsParams,
+} from '../types';
+
+type Props = { params: NotificationsParams; dropdownContent: DropdownsContent };
+
+export const NotificationsTable: FunctionComponent<Props> = ({
+    params,
+    dropdownContent,
+}) => {
+    const apiParams: ApiNotificationsParams = {
+        ...params,
+        limit: params.pageSize || '20',
+        order: params.order || 'id',
+        page: params.page || '1',
+    };
+    const { data, isFetching } = useGetNotifications(apiParams);
+    const { formatMessage } = useSafeIntl();
+    const columns: Array<Column> = [
```

Comment:
Not blocking but we usually extract that in a hook to void cluttering the table component